client.js

```diff
@@ -18,29 +18,92 @@ function createClientConfig() {
     }
 }
 
-(async () => {
-    try {
-        const client = await Client.newHazelcastClient(createClientConfig());
-        const map = await client.getMap('map');
-        await map.put("key", "value")
-        const res = await map.get("key")
-        if (res !== "value") {
-            throw new Error("Connection failed, check your configuration.")
-        }
-        console.log("Connection Successful!");
+function mapExample(map) {
```

Comment:
```suggestion
async function mapExample(map) {
```

you could make this function async and get rid of the one nesting below, by removing the async block.